WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml

```diff
@@ -121,13 +129,12 @@
             Style="{StaticResource GalleryTileGridStyle}">
             <ScrollViewer HorizontalScrollBarVisibility="Auto" HorizontalScrollMode="Auto">
                 <StackPanel Padding="12,24,12,12">
-                    <Grid HorizontalAlignment="Stretch">
+                    <Grid Margin="16,0,0,0" HorizontalAlignment="Stretch">
```

Comment:
Moving the margin was a bad change on my end. #oops

Reply:
Fixed it